2010-03-12  Ulrich Drepper  <drepper@redhat.com>

	* sysdeps/unix/sysv/linux/sys/mount.h (UMOUNT_NOFOLLOW): Define.

---
 sysdeps/unix/sysv/linux/sys/mount.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/sysdeps/unix/sysv/linux/sys/mount.h
+++ b/sysdeps/unix/sysv/linux/sys/mount.h
@@ -100,8 +100,10 @@
 #define MNT_FORCE MNT_FORCE
   MNT_DETACH = 2,		/* Just detach from the tree.  */
 #define MNT_DETACH MNT_DETACH
-  MNT_EXPIRE = 4		/* Mark for expiry.  */
+  MNT_EXPIRE = 4,		/* Mark for expiry.  */
 #define MNT_EXPIRE MNT_EXPIRE
+  UMOUNT_NOFOLLOW = 8		/* Don't follow symlink on umount.  */
+#define UMOUNT_NOFOLLOW UMOUNT_NOFOLLOW
 };
 
 
